lib/presentation/models/series_view_data.dart

```diff
@@ -1,9 +1,16 @@
-class SeriesViewData {
+import 'package:equatable/equatable.dart';
+
+class SeriesViewData extends Equatable {
   final String? title;
   final String? thumbnailUrl;
 
   const SeriesViewData({
     required this.title,
     required this.thumbnailUrl,
   });
+
+  @override
+  List<Object?> get props => [title, thumbnailUrl];
+
+
```

Comment:
Empty lines, in the first look it do not sounds important, but for the other developers it can say a lot, so I strongly suggest being more careful with the code that you are going to submit to review.